specs/0118-notification-center.md

```diff
@@ -0,0 +1,122 @@
+      - Select one
+      - Default value: `Select activity type`
+      - If one or more filters already exist, dynamically remove existing types from the dropdown list
+      - Form dynamically displays the appropriate filter inputs below based on the selection in the dropdown. `Select activity type` does not display any filter fields.
+      - Validation:
+        - Multiple activity filters can be used at the same time
+        - Disable `Review message` button and show error message if `Select activity type` selected as a filter in the dropdown: `You must select an activity type when you add an activity filter.`
+    - Filter inputs:
+      - Inputs for `Submitted translations` / `Performed reviews`
+        - `Activity level`
+          - Input box: `Minimum` (value required, default value: 0, only accepts non-negative integers)
+          - Input box: `Maximum` (value optional, default value: empty, leaving empty will mean there is no maximum and all numbers greater than or equal to the value in `Minimum` will be included.)
+          - Tooltip on `Maximum`: `Leave empty to include all numbers greater than or equal to Minimum.`
+        - `Timeframe filter`
+          - Checkbox (default: deselected): `Enable timeframe filter`
+            - Checking the box displays datepickers
+          - Datepicker: `From:` (Default value: 1 year before today's date)
+          - Datepicker: `To:` (Default value: today's date)
+      - Inputs for `Last login`
+        - `Timeframe filter`
+          - Datepicker: `From:` (Default value: today's date)
+          - Datepicker: `To:` (Default value: today's date)
+- Header: `Transactional email`
+  - By default, emails are sent to users who have enabled opt-in consent. (Does not apply to in-app notifications.)
+  - This “Transactional” option flags emails as transactional and sends emails to users regardless of their opt-in status. However, this email content would be restricted to content transactional in nature (e.g. account status notifications, etc.).
+  - Checkbox (default: deselected): `This is a transactional email`
+  - Text description: `Transactional emails are sent to users who have not opted in to email communication. Transactional emails are restricted in the type of content that can be included.`
+  - Validation:
+    - Disable `Review message` button and show error message if `Transactional email` enabled but `Message type` does not include `Email`: `You cannnot enable the transactional email option if the Message type does not include email.`
+- Button: `Review message`
+  - A send button is not displayed on the same page as the message editor and recipient selection to avoid sending an incomplete message or a message to the wrong recipients.
+  - When pressed a validation step is performed on inputs to ensure there are no issues with the form. If there are no issues, the user proceeds to the Review/Confirmation page.
+
+- Title: `Review message`
```

Comment:
Also, the list of bullet points in this section is very long and hard to follow. It should help a bit to split the Feature explanation into Compose and Review sections.